src/ralph/api/routers/statements.py

```diff
@@ -137,10 +139,12 @@ def strict_query_params(request: Request):
 
 @router.get("")
 @router.get("/")
-# pylint: disable=too-many-arguments, too-many-locals
 async def get(
     request: Request,
-    current_user: Annotated[AuthenticatedUser, Depends(get_authenticated_user)],
+    current_user: Annotated[
+        AuthenticatedUser,
+        Security(get_authenticated_user, scopes=["statements/read/mine"]),
```

Comment:
Following my previous comment, this may be expressed as:

```suggestion
        Security(get_authenticated_user, scopes=[SCOPES.STATEMENTS_READ_MINE]),
```